### PR TITLE
ts/momentum-indicator 

### PR DESCRIPTION
**Pull request by karolkolodziej**

### CODE REVIEW

File: js/masters/indicators/momentum.src.js [hunk 1]
```diff
@@ -11,4 +11,4 @@
  * License: www.highcharts.com/license
  */
 'use strict';
-import '../../Stock/Indicators/MomentumIndicator.js';
+import '../../Stock/Indicators//Momentum/MomentumIndicator.js';
```
Comment (bre1470) [inline js/masters/indicators/momentum.src.js:14]: Not sure if it is a problem on any platform, but to make sure the double slash should be fixed.

**Review by bre1470 — CHANGES_REQUESTED** [review]
Thank you! One small thing otherwise good.

File: js/masters/indicators/momentum.src.js (hunk 1 above)
Comment (karolkolodziej) [reply js/masters/indicators/momentum.src.js]: True! I missed that. Thanks for spotting that

**Review by bre1470 — APPROVED** [review]
Thank you! Looks good. 👍

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
What's up with the visual diff? It seems like the series name is suddenly all upper case, but I can't see any change in the code related to it...

File: ts/Stock/Indicators/Momentum/MomentumIndicator.ts [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+     * @optionparent plotOptions.momentum
+     */
+    public static defaultOptions: MomentumOptions = merge(SMAIndicator.defaultOptions, {
+        params: {
+            period: 14
+        }
+    } as MomentumOptions);
+
+    public data: Array<MomentumPoint> = void 0 as any;
+    public nameBase: string = void 0 as any;
```
Comment (bre1470) [inline ts/Stock/Indicators/Momentum/MomentumIndicator.ts:79]: `nameBase` musty be set like in the original file and goes on the prototype.

File: ts/Stock/Indicators/Momentum/MomentumIndicator.ts [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+/* *
+ *
+ *  Prototype Properties
+ *
+ * */
+
+interface MomentumIndicator {
+    pointClass: typeof MomentumPoint;
+}
+
```
Comment (bre1470) [inline ts/Stock/Indicators/Momentum/MomentumIndicator.ts:151]: Here the extend of prototype with properties has to be added:
```ts
extend(MomentumIndicator.prototype, {
    nameBase: 'Momentum'
});
```

**Review by bre1470 — CHANGES_REQUESTED** [review]
Spotted by @TorsteinHonsi , the baseName value is missing in the (missing) `extend` call.

File: ts/Stock/Indicators/Momentum/MomentumIndicator.ts (hunk 2 above)
Comment (bre1470) [reply ts/Stock/Indicators/Momentum/MomentumIndicator.ts:79]: To clarify it: `nameBase` has to be removed from the class pattern.

**Comment by TorsteinHonsi** [comment]
Pending @bre1470 's approval

**Review by bre1470 — APPROVED** [review]
Thank you! Looks good. 👍

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**